servicecontrol/installation.md

```diff
@@ -18,6 +18,8 @@ The ServiceControl Installation has the following prerequisites:
 
 If ServiceControl is installed via the Platform Installer then the installation and configuration of these prerequisites are managed by the installer.
 
+NOTE: We advise to install ServiceControl on a seperate (virtual) machine in a production environment to isolate the audit and error queue message ingestion. These operations have a big impact other other processes. Treat ServiceControl as you would setup a database server meaning sufficient CPU, RAM (6GM minimum), and storage suitable for low latency write operations.
```

Comment:
> advise to install ServiceControl on a seperate

advise installing ServiceControl on a separate

> These operations have a big impact other other processes

on other processes

Not really speaking to the needs of the database?

Might want to link to https://docs.particular.net/servicecontrol/capacity-and-planning ?